Plugin.pm

```diff
@@ -282,12 +282,58 @@ sub trackInfoMenu {
 	my $extid = $track->extid;
 	$extid ||= $url if $url =~ /^tidal:/;
 
-	return _objInfoMenu($client,
-		$extid,
-		$track->artistName || $remoteMeta->{artist},
-		$track->album ? $track->albumname : $remoteMeta->{album},
-		$track->title || $remoteMeta->{title},
-	);
+	my $artist = $track->remote ? $remoteMeta->{artist} : $track->artistName;
+	my $album  = $track->remote ? $remoteMeta->{album} : ( $track->album ? $track->album->name : undef );
```

Comment:
Did `$track->albumname` not work? that would make this line more readable.

Reply:
I can change that.  Don't know if that works for all cases though.  Why is it `$track->albumname` (all small letters) but `artistName` right above?